clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/MonitorFieldFilters.tsx

```diff
@@ -180,10 +180,15 @@ export const MonitorFieldFilters = ({
     setLocalDataCategory(dataCategory);
   }, [dataCategory]);
 
-  // Reset filters to default state when stagedResourceUrn changes
-  // Use JSON.stringify to compare array contents, not reference
+  // Reset filters to default state when stagedResourceUrn changes (but not on initial mount)
+  // Use join to compare array contents, not reference
   const stagedResourceUrnKey = stagedResourceUrn.join(",");
+  const isInitialMount = useRef(true);
   useEffect(() => {
+    if (isInitialMount.current) {
+      isInitialMount.current = false;
+      return;
+    }
```

Comment:
fixes a bug where reloading the page via browser would lose all currently selected filters/query params